inst/extdata/gcam-china/population_thous_province.csv

```diff
@@ -0,0 +1,37 @@
+# File: population_thous_province.csv
+# Title: Population by province
+# Units: 1000 persons
+# Description: Provincial population by historical and future years
+# Source: Citation or source if available
```

Comment:
data source needed @sypnl

Reply:
Historical data (1975-2013) on national population were obtained from the 2014 China Statistics Yearbook and the China Compendium of 60-year Statistics (1949-2008). UN medium variant growth rates were applied to project future population in China from 2014 to 2100.